service/README.md

```diff
@@ -53,17 +53,87 @@ Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/configurer/utils/fp
 the service.
 
 ### Run the application (from IntelliiJ)
- 
-Ensure the Spring Boot is started with local profile 
-(add environment variable spring.profiles.active=local when starting the main class).
 
-Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ 
-starts the application). 
+Ensure the Spring Boot is started with local, user-mappings and the feature-toggle profiles
+(add environment variable spring.profiles.active=feature-toggle,local,user-mappings when starting the main class).
 
-Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
-http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching 
-locally running application. 
+Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ
+starts the application).
 
+Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
+http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching
+locally running application.
+
+## Config
+
+Services uses three sources of configuration for the placeholders:
+* application.yaml (profile-driven, test and production systems use only default profile)
+* ENV variables (passed in via charts in the cluster or docker-compose for local)
+* Secrets from Azure Key vaults and following [library](https://github.com/hmcts/properties-volume-spring-boot-starter).
+  The secret names are map to the properties as specified in [this file](src/main/resources/bootstrap.yaml)
+
+Custom configuration parameters:
+
+|Property name|Configuration place|Description|
+|---|---|---|
+|fees-register.api.url|ENV|Address of the Fees Register service, used for obtaining fee of applications|
+|fees-register.parameters|application.yaml|Mapping of FPL's possible Fees to required parameters by Fee Register. Configuration stored in class *FeesConfig*|
+|payment.site_id|application.yaml|Identifier required when communicating with Payment service to determine payments coming from FPL |
+|payment.api.url|ENV|URL of service providing PBA payments functionality|
+|bankHolidays.api.url|application.yaml|URL of the service which can return this years Bank Holidays|
+|ld.sdk_key|SECRET|Launch Darkly API key for feature toggles|
+|ld.user_key|ENV|Optional parameter which is send to LD to enable per env toggles|
+|SCHEDULER_ENABLED|ENV|(Optional) Determines if Quartz scheduler should be used. Enabled by default (disabled in local profile).|
+|SCHEDULER_DB_HOST|ENV|(Optional) Host for Scheduler DB, default is: localhost|
```

Comment:
A bit of inconsistency again, either use the "default is: " on the others or change this one to
```suggestion
|SCHEDULER_DB_HOST|ENV|(Optional) Host for Scheduler DB, default is localhost|
```
Maybe even wrap the defaults in `` `...` ``

Reply:
Done